rest/src/request_types.py

```diff
@@ -2,6 +2,10 @@
 POST = 'POST'
 
 
+def dict_to_querystring(params):
+    return '&'.join([f'{k}={v}' for k, v in params.items()])
```

Comment:
not very understandable variable names

Reply:
actually it's quite common for short functions involving dictionaries to use `k` for key and `v` for value